src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -3,13 +3,18 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 
+import seedu.bankwithus.exceptions.CommandNotFoundException;
+
 public class BankWithUs {
 
     public static final String FILE_PATH = "data/save.txt";
 
+    public boolean isExitEntered = false;
```

Comment:
Public for convenience purposes. Since only parse user command will actually change the value of this variable